OpenRA.Mods.Common/Widgets/Logic/Ingame/IngameMenuLogic.cs

```diff
@@ -263,7 +263,7 @@ public static void OnQuit(World world)
 				Game.RunAfterDelay(exitDelay, () =>
 				{
 					if (Game.IsCurrentWorld(world))
-						mpe.Fade(MenuPostProcessEffect.EffectType.Black);
+						mpe.Fade(mpe.Info.GameExitEffect);
```

Comment:
It would be nice if we could decouple this bit of the UI from the specific trait implementation - time for a new interface notifying game quit / menu / restart?

Reply:
Perhaps, but it's not something I'm interested in doing